### DISCUSSION TITLE
Wysiwyg toolbar items ordering

### DISCUSSION THREAD

**Discussion by iksent** [directus/directus#4443] [Feature Requests]
Items in the toolbar are ordered as they are added, as a result, the order is not very nice, it will be cool to have an option to order them manually

![Снимок экрана 2021-03-09 в 11 32 45](https://user-images.githubusercontent.com/32956521/110428581-518f6300-80cb-11eb-8ddb-c8df0ac18e4b.png)

![Снимок экрана 2021-03-09 в 11 34 04](https://user-images.githubusercontent.com/32956521/110428616-5fdd7f00-80cb-11eb-8f9a-6d407d8b83d5.png)

**Comment by benhaynes** [comment]
I can't remember if the Dropdown component supports reordering, but that might be the way to go. Or a set order, where you enable/disable, but the button order uses a standard priority.

**Reply by djmtype** [reply]
It looks like it using `block_formats`. https://www.tiny.cloud/docs/configure/editor-appearance/#block_formats
Yes, please allow the formselect dropdown. 
Displaying the header tag buttons separately seems too cumbersome.

**Reply by HitomiTenshi** [reply]
Toolbar items are set in the order you check the checkboxes. This currently results in random button placements on various collections. I would love if this feature could be implemented! 😄 Even better would be a global option where you can set the default WYSIWYG toolbar items and order so you don't have to redo it on every WYSIWYG field you add. 👍

**Reply by benhaynes** [reply]
We might need to use a different interface for configuring these buttons, but I agree. Either the order should be set by default (the native TinyMCE order) or we need to allow explicit sorting in the config.

**Reply by rijkvanzanten** [reply]
Tracked that as an issue here: https://github.com/directus/directus/issues/10639

It should at least adhere to the pre-defined list of the dropdown. After that, we can look into potentially improving with a custom order (assuming it meets 80/20)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
